docs/docs/developers/guides/js_apps/use_fee_juice.md

```diff
@@ -71,20 +115,28 @@ import { CLI_Fees } from '/components/snippets';
 
 <CLI_Fees />
 
-See here to [Bridge Fee Juice](../../../developers/reference/environment_reference/cli_wallet_reference#bridge-fee-juice) via the CLI wallet.
+### Claim and deploy
 
-### Claim and pay
+Here we will use the account managers and wallets of `myAddresses` to create the payment method, and use it for accounts to claim fee juice and pay for their own deployment.
 
-After a user has sent fee juice from L1 to be bridged to L2, a transaction can be made that claims this to pay for itself, and make the transaction, in one.
+Note: This uses the `claim` object shown in the earlier bridging section.
 
-The claim object:
+```ts
+// claim and pay to deploy accounts
+let sentTxs = [];
```

Comment:
This snipped could be simplified by showing it happen with 1 tx instead of an array of txs. And can we link to source code instead of hardcoding the snippet?